README.md

```diff
@@ -1,19 +1,16 @@
-### hey, I'm Molim 👋
-
-const molim = {
+# Hey, I'm Molim 👋
```

Comment:
Like in HTML, it's a good practice to have an "h1" in "your page".

If you don't like the styles you can change it back or set an `h2` (`##`)